fenix/app/src/main/java/org/mozilla/fenix/library/bookmarks/BookmarkItemMenu.kt

```diff
@@ -36,7 +36,7 @@ class BookmarkItemMenu(
     @VisibleForTesting
     @SuppressWarnings("LongMethod")
     internal suspend fun menuItems(itemType: BookmarkNodeType, itemId: String): List<TextMenuCandidate> {
-        val hasAtLeastOneChild = !context.bookmarkStorage.getTree(itemId)?.children.isNullOrEmpty()
+        val hasAtLeastOneChild = !context.bookmarkStorage.getTree(itemId, false)?.children.isNullOrEmpty()
```

Comment:
This isn't technically part of the home activity changes, but while looking for other suspect uses of `getTree` I discovered this, which doesn't actually need the full content of each child but will work just fine with just the IDs of the children.